chapter2/tests/t20/terraform/dev/passwords.tf

```diff
@@ -0,0 +1,89 @@
+resource "random_password" "wordpress_password" {
```

Comment:
move these random resources to concrete tf files where they are being invoked/used.